src/config/config.cc

```diff
@@ -657,30 +661,40 @@ Status Config::finish() {
   return Status::OK();
 }
 
-Status Config::Load(const std::string &path) {
-  if (!path.empty()) {
-    path_ = path;
-    std::ifstream file(path_);
-    if (!file.is_open()) return Status(Status::NotOK, strerror(errno));
+Status Config::Load(const CLIOptions &opts) {
+  if (!opts.conf_file.empty()) {
+    std::ifstream file;
+    std::istream *in = nullptr;
+    if (opts.conf_file == "-") {
+      in = &std::cin;
+    } else {
+      path_ = opts.conf_file;
+      file.open(path_);
```

Comment:
iostream will close the file after exiting the scope, right?

Reply:
Yeah

Reply:
Thanks~